test/test_onnx.py

```diff
@@ -99,6 +100,17 @@ def forward(self, boxes, scores):
 
         self.run_model(Module(), [(boxes, scores)])
 
+    def test_clip_boxes_to_image(self):
+        boxes = torch.randint(10, (5, 4))
+        boxes[:, 2:] += torch.randint(500, (5, 2))
```

Comment:
FYI this doesn't guarantee that the boxes are wellformed, because the second coordinate can be smaller than the first coordinate.
A safer implementation would be something like
```python
boxes = torch.randint(...)
boxes[:, 2:] += boxes[:, :2]
```

Reply:
Thanks. I actually changed it to boxes[:, 2:] += boxes[:, :2] + torch.randint(500, (5, 2)) to get larger dimensions in last two coordinates for clamp.